acceptance/tests/group/group_local_unicode_group.rb

```diff
@@ -3,22 +3,20 @@
 confine(:to, :platform => 'windows')
 
 #Globals
-user_type = 'local_unicode_group'
+prefix = SecureRandom.uuid.to_s
 file_content = 'Dangers driving drunk while insane.'
 
 parent_name = 'temp'
-target_name = "group_#{user_type}.txt"
+target_name = "#{prefix}.txt"
 
 target_parent = "c:/#{parent_name}"
 target = "#{target_parent}/#{target_name}"
 user_id = 'bob'
-group_id = '䎈含㴼罍率䎁叴秀㪲軞'
 
-verify_content_command = "cat /cygdrive/c/#{parent_name}/#{target_name}"
-file_content_regex = /\A#{file_content}\z/
+raw_group_id = 'group_\u4388\u542B\u3D3C\u7F4D\uF961\u4381\u53F4\u79C0\u3AB2\u8EDE'
+group_id =     "group_\u4388\u542B\u3D3C\u7F4D\uF961\u4381\u53F4\u79C0\u3AB2\u8EDE" # 䎈含㴼罍率䎁叴秀㪲軞
```

Comment:
Not necessary to change here, but should you need it, you can go from string to Unicode escape sequence like:

```ruby
str.unpack('U*').map{ |i| "\\u" + i.to_s(16).rjust(4, '0') }.join
```

Note that you need a little more code to not escape the normally printable everywhere 0-127 ASCII range.

Reply:
I did thank about trying to codify it, but I resorted to just duplicating it as it's easier to read.